organization/views.py

```diff
@@ -202,7 +187,7 @@ def get_reference_organization(self):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        context['basics'] = True
+        context['is_basics_active'] = True
```

Comment:
I like this solution a lot! I think there are ways we could modularize it even further. Since the logic for all the views is so similar, one way to encapsulate this behavior would be to create a mixin providing a `get_context_data` method that all of the views could inherit from -- something like:

```python
class NavTabMixin:
    def get_context_data(self, **kwargs):
        context = super().get_context_data(**kwargs)
        context['is_%s_active' % self.nav_tab] = True
        return context
```

Then, to graft the behavior onto a view, all you would have to do would be to have it inherit the mixin and set a value for `self.nav_tab`, e.g.:

```diff
-class OrganizationEditBasicsView(OrganizationEditView):
+class OrganizationEditBasicsView(NavTabMixin, OrganizationEditView):
     template_name = 'organization/edit-basics.html'
     form_class = OrganizationBasicsForm
+    nav_tab = 'basics'
```

Optionally, if it seems like we want this behavior on every single `OrganizationEdit` and `OrganizationCreate` view, we could also think about moving the `get_context_data` method on `NavTabMixin` above to parent classes, such as `OrganizationEditView`, to take advantage of class inheritance instead of the composition of a mixin. This would be a larger refactor, since we would need to create an `OrganizationCreateView` parent class to match `OrganizationEditView`, but if that can be done reasonably then it would also be a pretty high-value refactor in my opinion.

How does that sound to you? Consider the code above to be a suggestion of what the code could look like, not a strict guideline for how I expect it to be implemented.

Reply:
i decided to go with the mixin for ease of implementation and (mostly) speed.

Reply:
Sounds good! We can always refactor to use inheritance later.